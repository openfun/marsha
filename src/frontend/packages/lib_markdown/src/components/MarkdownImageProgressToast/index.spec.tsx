import { render } from 'lib-tests';
import React from 'react';

import { MarkdownImageProgressToast } from '.';

describe('<MarkdownImageProgressToast />', () => {
  it('displays progression', () => {
    const { container } = render(
      <MarkdownImageProgressToast filename="some-file.test" progress={80} />,
    );

    expect(container).toMatchSnapshot();
  });
});
