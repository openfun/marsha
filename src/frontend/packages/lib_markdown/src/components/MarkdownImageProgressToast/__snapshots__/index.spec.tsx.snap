// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MarkdownImageProgressToast /> displays progression 1`] = `
<div>
  <div
    class="StyledGrommet-sc-19lkkz7-0 bglOOo"
  >
    <div
      style="position: fixed; z-index: 9999; top: 16px; left: 16px; right: 16px; bottom: 16px; pointer-events: none;"
    />
    <div
      data-testid="test-component-container"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 QcCYf"
      >
        <span
          class="StyledText-sc-1sadyjn-0 lhOBjf"
        >
          some-file.test
        </span>
        <div
          class="StyledBox__StyledBoxGap-sc-13pk1d4-1 ejkSNv"
        />
        <div
          class="StyledBox-sc-13pk1d4-0 hoFOaz"
        >
          <div
            class="StyledStack-sc-ajspsk-0 bkMvqh"
          >
            <div
              class="StyledStack__StyledStackLayer-sc-ajspsk-1 gLEcPM"
            >
              <svg
                class="StyledMeter-sc-nsxarx-0 fVMZrb"
                height="24"
                preserveAspectRatio="none"
                viewBox="0 0 384 24"
                width="384"
              >
                <path
                  d="M 0,12 L 384,12"
                  fill="none"
                  stroke="#eeeeef"
                  stroke-linecap="square"
                  stroke-opacity="0.4"
                  stroke-width="24"
                />
                <path
                  d="M 0,12 L 307.2,12"
                  fill="none"
                  stroke="#007bff"
                  stroke-linecap="butt"
                  stroke-width="24"
                />
              </svg>
            </div>
            <div
              class="StyledStack__StyledStackLayer-sc-ajspsk-1 kfgmlQ"
            >
              <span
                class="StyledText-sc-1sadyjn-0 hUfGMB"
              >
                80%
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
